fake_switches/switch_configuration.py

```diff
@@ -101,6 +112,20 @@ def __init__(self, name):
         self.name = name
 
 
+class Route(object):
+    def __init__(self, prefix, mask, ip):
```

Comment:
For all naming conventions, I would suggest using the following:
prefix --> Destination network --> (simply destination? ou network_ip?)
mask --> Destination mask --> (simply mask?)
ip --> Next Hop --> (simply next_hop?)

it would be more accurate since 'prefix' isn't much of a thing in networking but a mask (and it's still called a mask....)